OneMore/OneNote.cs

```diff
@@ -195,13 +195,30 @@ public void Dispose()
 		/// </summary>
 		public string CurrentNotebookId => onenote.Windows.CurrentWindow?.CurrentNotebookId;
 
-
 		/// <summary>
-		/// Gets or sets whether exceptions are allowed to fall through back to caller or
-		/// are caught and reported by this class. This is a special case for some consumers
-		/// who wish to handle certain exceptions themselves to serve data management.
+		/// Gets all the page structure
 		/// </summary>
-		public bool FallThrough { get; set; }
+		//public string NotebookXml => GetAllPages(); //onenote.Windows.CurrentWindow?.CurrentNotebookId;
+
+        public XElement GetAllPages()
+        {
+            onenote.GetHierarchy(null, HierarchyScope.hsPages, out string xml, XMLSchema.xs2013);
+            if (!string.IsNullOrEmpty(xml))
+            {
```

Comment:
There's not a lot of value in this helper function beyond what GetHierarchy already provides. I'd suggest just inlining it within your business logic rather than adding to the OneNote class.